src/core/models/outputs/native_tokens.h

```diff
@@ -9,67 +9,75 @@
 
 #include "core/types.h"
 #include "core/utils/uint256.h"
-#include "uthash.h"
+#include "utlist.h"
 
 // Native Token ID length in bytes
 #define NATIVE_TOKEN_ID_BYTES 38
 // Serialized bytes = token ID(38 bytes) + amount(uint256_t)
 #define NATIVE_TOKENS_SERIALIZED_BYTES (NATIVE_TOKEN_ID_BYTES + 32)
 
 /**
- * @brief Native Tokens is a set of Native Token
+ * @brief Native Token structure
  *
  */
 typedef struct {
   byte_t token_id[NATIVE_TOKEN_ID_BYTES];  ///< Identifier of the Native Token
   uint256_t *amount;                       ///< Amount of Tokens. Pointer to uint256_t object
```

Comment:
the amount is mandatory, it can stay in the stack and prevent memory fragmentation.
```suggestion
  uint256_t amount;                       ///< Amount of the Native Token
```

Reply:
Thanks. Fixed.